src/nvim/charset.c

```diff
@@ -734,7 +734,7 @@ int vim_strnsize(char_u *s, int len)
 /// @return Number of characters.
 #define RET_WIN_BUF_CHARTABSIZE(wp, buf, p, col) \
   if (*(p) == TAB && (!(wp)->w_p_list || lcs_tab1)) { \
-    int ts; \
+    int64_t ts; \
```

Comment:
Well, it is a save of an `long` variable (and would be changed along with it, well, because -Wconversion :), used in an `int` context, so it should be one of those two. But you're right `(int)` is better :)